modules/local/prokka.nf

```diff
@@ -0,0 +1,50 @@
+process PROKKA {
```

Comment:
Why aren't you using the nf-core module, plus a patch for the changes you made?

Reply:
Yeah, that is the general rule I would say. For we don't have a prokka nf-module so in that case we should use the nf-core one.